app/controllers/games_controller.rb

```diff
@@ -0,0 +1,35 @@
+class GamesController < ApplicationController
+  before_action :authenticate_user!, only: [:new, :create, :update]
+
+  def new
+    @game = Game.new
+  end
+
+  def create
+    @game = Game.create(game_params.merge(white_player_id: current_user.id, state: "open"))
+    if @game.persisted?
+      redirect_to game_path(@game)
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def show 
+    @game = Game.find(params[:id])
+    rescue ActiveRecord::RecordNotFound
+    render text: 'Game Not Found', status: :not_found
+  end
+
+  def update 
+    @game = Game.find(params[:id])
```

Comment:
Optional: A general principle that I like to adhere to is making data and methods only available to the parts of the app that need them. In this case, since you're not trying to relay data to a view (the normal use for instance variables in controllers), you can make do with just a local version: `game`.